src/Umbraco.Infrastructure/Migrations/Upgrade/V_9_2_0/MovePackageXMLToDb.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using Umbraco.Cms.Core.Packaging;
+using Umbraco.Cms.Infrastructure.Persistence;
+using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_2_0
+{
+    public class MovePackageXMLToDb : MigrationBase
+    {
+
+        private readonly PackagesRepository _packagesRepository;
```

Comment:
Shouldn't we use here `CreatedPackageSchemaRepository` since we obsoleted `PackagesRepository`?

Reply:
This is because we need the old packages repository to run the migration 🙈 
The problem I ran into is, I have to somehow get all the packages that are currently saved(either by searching directly in the filesystem, or using the old repo) so we have to use the old repo to get all the currently saved files and put them in the database